### PR TITLE
ADCM-1979 pylint settings split 

### PR DESCRIPTION
**Pull request by skhomuti**
Also make linters faster

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,10 +78,13 @@ ng_tests: ## Run Angular tests
...
 linters : ## Run linters
 	docker pull hub.adsw.io/library/pr-builder:3-x64
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh shellcheck pylint pep8
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests black
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests/functional flake8_pytest_style
-	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 /linters.sh -f ./tests/ui_tests flake8_pytest_style
+	docker run -i --rm -v $(CURDIR)/:/source -w /source hub.adsw.io/library/pr-builder:3-x64 \
+        /bin/bash -xeo pipefail -c "/linters.sh shellcheck pylint pep8 && \
+        /linters.sh -b ./tests -f ui_tests pylint && \
+        /linters.sh -b ./tests -f functional pylint && \
```
Comment (dgusakov) [inline Makefile:84]: I guess we should lint all tests folder here

File: Makefile [hunk 2]
```diff
@@ -78,10 +78,13 @@ ng_tests: ## Run Angular tests
...
+        /linters.sh -f ./tests black && \
+        /linters.sh -f ./tests/functional flake8_pytest_style && \
+        /linters.sh -f ./tests/ui_tests flake8_pytest_style"
```
Comment (dgusakov) [inline Makefile:87]: Same here

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (skhomuti) [reply Makefile:84]: unfortunately it doesn't work yet
```
************* Module ui_tests.test_cluster_list_page

ui_tests/test_cluster_list_page.py:46: [E0401(import-error), ] Unable to import 'ui_tests.utils'
```

File: tests/ui_tests/test_host_page.py [hunk 3]
```diff
@@ -276,11 +270,7 @@ def test_open_host_from_host_list(
 
 
 @pytest.mark.usefixtures("_create_host")
-def test_delete_host(
-    sdk_client_fs: ADCMClient,
-    page: HostListPage,
-    upload_and_create_provider: Tuple[Bundle, Provider],
-):
+def test_delete_host(page: HostListPage):
```
Comment (Sealwing) [inline tests/ui_tests/test_host_page.py:273]: Add `upload_and_create_provider` to usefixtures

**Review by Sealwing — CHANGES_REQUESTED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**